dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/process/ProcessService.java

```diff
@@ -599,6 +599,26 @@ private ProcessInstance generateNewProcessInstance(ProcessDefinition processDefi
         return processInstance;
     }
 
+    private void setGlobalParamIfCommanded(ProcessDefinition processDefinition, Map<String, String> cmdParam) {
+        // get start params from command param
```

Comment:
Thank you very much for your contribution, but I noticed that this piece of code is also used elsewhere (generateNewProcessInstance), and it is best to modify it.